eng/native/configurecompiler.cmake

```diff
@@ -141,6 +141,20 @@ elseif (CLR_CMAKE_HOST_UNIX)
 endif(MSVC)
 
 if (CLR_CMAKE_ENABLE_SANITIZERS)
+  enable_language(C)
```

Comment:
Is it necessary? IOW, are there cases where C not enabled and control reaches this line?